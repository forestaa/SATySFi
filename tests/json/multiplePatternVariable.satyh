let m = match (1, 2) with
  | (m, m) -> 0
