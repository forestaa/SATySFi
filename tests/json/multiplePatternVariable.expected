{
  "file": "multiplepatternvariable.satyh",
  "range": {
    "start": { "line": 2, "character": 5 },
    "end": { "line": 2, "character": 9 }
  },
  "message": "! [Type Error] pattern variable 'm' is bound more than once."
}
